responsive-web-design/css-typography/index19.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Nutrition Label</title>
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,700,800" rel="stylesheet">
+  </head>
+  <style>
+    html {
+      box-sizing: border-box;
+    }
+
+    *, *:before, *:after {
+      box-sizing: inherit;
+    }
+    
+    body {
+      font-family: 'Open Sans', sans-serif;
+      font-size: 16px;
+    }
+
+    .label {
+      border: 2px solid black;
+      width: 270px;
+      margin: 20px auto;
+      padding: 0 7px;
+    }
+
+    h1 {
+      font-weight: 800;
+      text-align: center;
+      margin: -4px 0;
+    }
+
+    p {
+      margin: 0;
+    }
+  </style>
+  <body>
+    <div class="label">
+      <h1>Nutrition Facts</h1>
+      <p>8 servings per container</p>
+      <p>Serving size 2/3 cup (55g)</p>
+    </div>
+  </body>
+</html>
+
+<!--
+Lines can help separate and group important content, especially when space is limited.
+Add a div element with the class "divider" under the h1 element. The div should be empty, so don't put any text between the opening and closing div tags.
+Here's an example of an empty div element with a different classname: `<div class="test"></div>`
```

Comment:
I may be wrong about this, but if an element is not going to have any children could you use a self closing tag such as `<div class="divider" />`?

Reply:
I thought so too originally. It turns out that `<div>` is not self closing, and neither is `<span>`. When I first built the project I used `<hr>` which doesn't need a closing tag, but the spec changed and it's meant to be used as a "thematic break between paragraph level elements". Might look a bit cleaner without all the closing tags, though. What do you think?